server-src/src/main/java/com/itt/kmt/jwt/exception/ExceptionAdvice.java

```diff
@@ -59,6 +59,6 @@ public ModelMap handle401() {
     @ResponseStatus(HttpStatus.BAD_REQUEST)
     public ModelMap globalException(final HttpServletRequest request, final Throwable ex) {
         log.debug("Failed due to exception : " + ex.getMessage());
-        return new ModelMap().addAttribute("success", new ResponseMsg(Boolean.FALSE, Constants.BAD_REQUEST_MSG));
+        return new ModelMap().addAttribute("success", new ResponseMsg(Boolean.FALSE, ex.getMessage()));
```

Comment:
I suspect unit test cases might get fail due to this
Please cross check once

Reply:
Thanks Rohan, updated the same in unit test case